extras/userdoc/md/documentation/documentation.md

```diff
@@ -1,30 +1,25 @@
-Documentation
-=============
+# Documentation
 
-[Installing NEST](installation.md "Installation")
------------------------------------------------------------------------------
+## [Installing NEST](installation.md "Installation")
 
-[Introduction to PyNEST](introduction-to-pynest.md "Introduction to PyNEST")
---------------------------------------------------------------------------------------------------------
+## [Introduction to PyNEST](introduction-to-pynest.md "Introduction to PyNEST")
 
-NEST User Manual
----------------------------------------------------------------------------
+## NEST User Manual
 
--   [Frequently Asked Questions](frequently_asked_questions.md "Frequently Asked Questions")
--   [Analog recording with multimeter](analog_recording_with_multimeter.md "Analog recording with multimeter")
--   [Connection Management](connection_management.md "Connection Management")
--   [Topological Connections](wp-content/uploads/2015/04/Topology_UserManual.pdf "Topological Connections")
--   [Random numbers in NEST](random_numbers.md "Random numbers in NEST")
--   [Parallel Computing](parallel_computing.md "Parallel Computing")
+-   [Frequently Asked Questions](frequently-asked-questions.md "Frequently Asked Questions")
+-   [Analog recording with multimeter](analog-recording-with-multimeter.md "Analog recording with multimeter")
+-   [Connection Management](connection-management.md "Connection Management")
+-   [Topological Connections](http://www.nest-simulator.org/wp-content/uploads/2015/04/Topology_UserManual.pdf "Topological Connections")
```

Comment:
@heplesser: shouldn't we convert the topology manual to an IPython notebook or a plain markdown document at one point? I think the days of PDFs for documentation are over ;-) Can you note this somewhere?